locale/gu.yml

```diff
@@ -1700,346 +1700,415 @@ abs: "એબીએસ"
   # ❓ @sshiv5768 I think "સંપૂર્ણ" fits better for abs, which returns absolute value.
   # 👀 @136tejas
   # 👀 @Nidhir2k1
+  # ✅  @nik132-eng
 all: "બધા"
   # ✅ @bvishal8510
   # ✅ @sshiv5768
   # 👀 @136tejas
   # ✅ @Nidhir2k1
+  # ✅  @nik132-eng
 any: "કોઈપણ"
   # ✅ @bvishal8510
   # ✅ @sshiv5768
   # 👀 @136tejas
   # ✅ @Nidhir2k1
+  # ✅  @nik132-eng
 ascii: "ascii"
```

Comment:
@nik132-eng @madiedgar 
Instead of using the same ENGLISH word itself, when the word is an abbreviation, can we either use its pronunciation in the local language or spell it out in local translation. 
For eg. ascii here can either be અસકીઈ(pronunciation-wise) or એ-એસ-સી-આઈ-આઈ  (spell-out translations)

If we follow one such suggested approach, we will be able to keep the objective of this project intact.
Waiting for your suggestions.

Reply:
The above-suggested review is applicable for all such repetitive instances.

Reply:
yes, we can,
I was confused at that time if @madiedgar  also agrees with this suggestion I can change all values in Gujrati.

Reply:
Wonderful point @shahkv95! More and more individuals in other languages have been suggesting to spell it out in the local translation. I'd prefer as little english as possible, but I am deferring to you all for what you believe will be best in the context of your language. 

@nik132-eng I will merge this PR now (just to be sure you get the Hacktoberfest credit), but feel free to go back and update your own translation reviews to reflect the idea above. 👍